trello_functions.py

```diff
@@ -0,0 +1,34 @@
+import requests
+from item import Item
+
+auth_params = {'key' : 'b7ff82f3970592f40825cfb7c771881c', 'token' : 'b3918107d6c9333fcc87a9b0a362b1801db711b71559f152a724b63d5b070b62'}
+board_id = '5efc773fd08d053db4ef3c18'
```

Comment:
Like all the other trello ids here, they shouldn't be hardcoded (or at least mentioned in the README as values that need to be swapped out).

Probably worth refactoring out a config class at the very least.